DirectProgramming/C++SYCL_FPGA/Tutorials/Features/multi_aocx/device_code_split/README.md

```diff
@@ -0,0 +1,247 @@
+   ```
+
+   Alternatively, to compile for the Intel&reg; FPGA PAC D5005 (with Intel Stratix&reg; 10 SX), run `cmake` using the command:
+
+   ```
+   cmake -G "NMake Makefiles" .. -DFPGA_DEVICE=intel_s10sx_pac:pac_s10
+   ```
+
+   You can also compile for a custom FPGA platform. Ensure that the board support package is installed on your system. Then run `cmake` using the command:
+
+   ```
+   cmake -G "NMake Makefiles" .. -DFPGA_DEVICE=<board-support-package>:<board-variant>
+   ```
+
+2. Compile the design through the generated `Makefile`. The following build targets are provided, matching the recommended development flow:
+
+   - Compile for emulation (fast compile time, targets emulated FPGA device):
+
+     ```
+     nmake fpga_emu
+     ```
+
+   - Generate the optimization report:
+
+     ```
+     nmake report
+     ```
+
+   - Compile for simulation (fast compile time, targets simulated FPGA device, reduced problem size):
+
+     ```
+     nmake fpga_sim
+     ```
+
+   - Compile for FPGA hardware (longer compile time, targets FPGA device):
+
+     ```
+     nmake fpga
+     ```
+
+> **Note**: The Intel&reg; PAC with Intel Arria&reg; 10 GX FPGA and Intel&reg; FPGA PAC D5005 (with Intel Stratix&reg; 10 SX) do not yet support Windows*. Compiling to FPGA hardware on Windows* requires a third-party or custom Board Support Package (BSP) with Windows* support.
+
+> **Note**: If you encounter any issues with long paths when compiling under Windows*, you might have to create your `build` directory in a shorter path, for example `c:\samples\build`.  You can then run `cmake` from that directory, and provide `cmake` with the full path to your sample directory.
+
+### In Third-Party Integrated Development Environments (IDEs)
+
+You can compile and run this tutorial in the Eclipse*IDE (in Linux*) and the Visual Studio*IDE (in Windows*).
+For instructions, refer to [FPGA Workflows on Third-Party IDEs for Intel&reg; oneAPI Toolkits](https://www.intel.com/content/www/us/en/developer/articles/technical/intel-oneapi-dpcpp-fpga-workflow-on-ide.html).
+
+## Examining the Reports
+
+Locate the pair of `report.html` files in either:
+
+- **Report-only compile**:  `device_code_split_report.prj`
+- **FPGA hardware compile**: `device_code_split.prj`
+
+Scroll down on the Summary page of the report and expand the section titled 
+
+## Running the Sample
+
+1. Run the sample on the FPGA emulator (the kernel executes on the CPU):
+
+   ```bash
+   ./device_code_split.fpga_emu    (Linux)
+   device_code_split.fpga_emu.exe  (Windows)
+   ```
+
+2. Run the sample of the FPGA simulator device
+
+   ```bash
+   ./device_code_split.fpga_sim        (Linux)
+   device_code_split.fpga_sim.exe      (Windows)
+   ```
+
+3. Run the sample on the FPGA device
+
+   ```bash
+   ./device_code_split.fpga             (Linux)
+   device_code_split.fpga.exe           (Windows)
+   ```
+
+### Example of Output on Emulator
+
+
+### Discussion of Results
```

Comment:
You can show the difference in area utilization in the reports and show how per_kernel generated image uses lesser area than per_source if the tutorial is doing both compiles.